### PR TITLE
Rename cbind to cmapM - #38

### LINKED ISSUE #38: Rename `cbind` to `cmapM`

### PR DESCRIPTION
**Pull request by Cmdv**
fixes #38

### CODE REVIEW

**Comment by Cmdv** [comment]
@vrom911 ah yes not a problem I'll add those now 👍

File: co-log-core/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ Change log
 `co-log-core` uses [PVP Versioning][1].
 The change log is available [on GitHub][2].
 
+0.0.0.2
```
Comment (vrom911) [inline co-log-core/CHANGELOG.md]: This should also be `0.1.0` to match what we have in the `.cabal` (the real version 🙂)
Also, as `0.0.0.1` is not released you can put all the changes from there (which are in the below section) in `0.1.0` as well.

**Comment by Cmdv** [comment]
semver always catches me out!! but think it should be ok now 😄

**Comment by Cmdv** [comment]
whoop one step at a time 🎉

**Pull request merged**